docker/envoy-gateway/Dockerfile

```diff
@@ -1,8 +1,7 @@
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM gcr.io/distroless/static:nonroot
-ARG TARGETPLATFORM
-COPY $TARGETPLATFORM/envoy-gateway /usr/local/bin/
+COPY ./envoy-gateway /usr/local/bin/
```

Comment:
lets revert this, we only want to copy the binary tied to the arch, for relevance as well as image size

Reply:
Thanks, but it does tie to the arch, we do not need to add this ARG, we can control it by commands.
When make image, we build binary in target platform, and build image with the built binary, the arch is synced to be the same.